app.rb

```diff
@@ -35,7 +34,8 @@ def main_selection
     when '4' then create_book
     when '5' then create_rental
     when '6' then list_all_rentals_by_id
-    when '7' then abort('Thanks for your business. Goodbye!')
+    # when '7' then abort('Thanks for your business. Goodbye!')
+    # Can't add line 7 because rubocop complains about cyclomatic complexity :-(
```

Comment:
- [ ] Kindly find a way to fix this so as to maintain the functionality of the app because right now a user cannot exit your console app 👍

Reply:
Hi @Kingstalux thanks for your feedback. Do you have any suggestions on how to lower the cyclomatic complexity issue for the case statement? I've researched about this for a while and most suggest to disable the rubocop warning, but that will give a linter error.
What would you kindly suggest?